tests/python/test_python_frontend.py

```diff
@@ -3881,7 +3903,7 @@ def fusion_func(fd: FusionDefinition) -> None:
 
             fd.add_output(T88)
 
-        nvf_out, _ = self.exec_nvfuser(fusion_func, inputs, is_clonable=True)
```

Comment:
IIUC, the default behavior is `is_clonable = True`, so this is just a clean up?!